src/state/wallets/selectors.ts

```diff
@@ -54,7 +54,7 @@ export const transactions = createSelector(wallets, electrumXSelectors.blockHeig
   flatten(
     walletsList.filter(negate(isAllWallets)).map(wallet => {
       const walletBalanceUnit = wallet.getPreferredBalanceUnit();
-      const walletLabel = wallet.getLabel();
+      const walletLabel = wallet.label !== CONST.allWallets ? wallet.getLabel() : 'All wallets';
```

Comment:
there shouldn't be a need for this condition as we negate(isAllWallets) in the filter before maping

Reply:
we now working with all wallets and we need that for no error in getLabels

Reply:
but when this can occur if we have condition `walletsList.filter(negate(isAllWallets))`? And I see that even this filter seems to be not needed as `wallets` selector doesn't include AllWallets

Reply:
ok rewrite